### ISSUE TITLE
Port to Python 3

### ISSUE THREAD

**Issue by quozl** [sugarlabs/cookie-search-activity#19]

**Comment by Saumya-Mishra9129** [comment]
Hey!! I want to work on this issue.

**Comment by shivamrai1904** [comment]
Hey!! i want to work on this issue.

**Comment by Saumya-Mishra9129** [comment]
@shivamrai1904 Please have a look at the current pull requests, and try to figure out what work has been done so far, and what else has been left to do for a successful Python3 port of activity. As far I know in latest pull request, work for port has been done so, You can review it once. Also #18  is the major obstacle and needs to fix.

**Comment by daredevil3435** [comment]
Is this issue still opne?

**Comment by quozl** [comment]
Yes.

**Comment by daredevil3435** [comment]
> Yes.

Can you explain more?

**Comment by quozl** [comment]
Not really, not without detailed questions from you.  What has been said already in this issue (scroll up) and the currently open pull requests is an accurate assessment of the situation.  What have you done so far?
